all/test/test.proto

```diff
@@ -5,6 +5,7 @@ package Messages;
 import "all/test/include.proto";
 import "google/api/annotations.proto";
 import "google/protobuf/field_mask.proto";
+import "all/test/validate.proto";
```

Comment:
```suggestion
import "validate/validate.proto";
```

The file is included in the docker image (in /opt/include/validate/validate.proto) so you don't need to add it to the repo.

Reply:
took care of it